quarkus/src/main/resources/db/migration/V0.1__initial_schema.sql

```diff
@@ -122,5 +147,18 @@ create table order_details
 alter table ${flyway:defaultSchema}.order_details
     owner to ${flyway:user};
 
-create sequence hibernate_sequence start 1 increment 1;
+create table item_stock_reservation
```

Comment:
I think you should also try to create the table within the schema: `${flyway:defaultSchema}.item_stock_reservation`
If you change the `defaultSchema` variable, then it would: create a table in the database default schema and try to alter the table with changed schema, based on that variable.

Reply:
sure!